projects/packages/jetpack-mu-wpcom/tests/php/features/launchpad/class-launchpad-task-lists-test.php

```diff
@@ -22,4 +22,86 @@ public function test_build_creates_no_PHP_warnings() {
 		$this->assertIsArray( $result );
 		$this->assertEmpty( $result );
 	}
+
+	/**
+	 * Callback to check if the first task in a task list is completed.
+	 *
+	 * @param array $task_list Task list.
+	 */
+	public function callback_first_task_completed( $task_list ) {
+		$first_task = reset( wpcom_launchpad_checklists()->build( $task_list['id'] ) );
+		return $first_task['completed'];
+	}
+
+	/**
+	 * Data provider for ::test_task_list_is_completed().
+	 */
+	public function provider_test_task_list_is_completed() {
+		$incomplete_task = array();
+		$completed_task  = array(
+			'is_complete_callback' => '__return_true',
+		);
+
+		$launch_task = array(
```

Comment:
I think it would be easier to understand the setup if you included `incomplete` in the variable name.
```suggestion
		$launch_task_incomplete = array(
```